crates/rattler_solve/src/libsolv/wrapper/repo.rs

```diff
@@ -94,6 +94,7 @@ impl RepoRef {
         let mut package_to_type: HashMap<&str, (PackageExtension, SolvableId)> = HashMap::new();
```

Comment:
I think you don't need the  type here right?

Reply:
Unfortunately, the code doesn't compile without this here.